dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/category/CategoryOptionComboServiceTest.java

```diff
@@ -99,7 +103,9 @@ public void setUpTest()
         categoryService.addCategoryOption( categoryOptionD );
 
         categoryA = new Category( "Gender", DataDimensionType.DISAGGREGATION );
+        categoryA.setShortName( categoryA.getName() );
         categoryB = new Category( "Agegroup", DataDimensionType.DISAGGREGATION );
+        categoryB.setShortName( categoryB.getName() );
```

Comment:
Maybe use `DhisConvenienceTest#createCategory` here while we are at it?

Reply:
Can't. That method does not allow any name. We might want to extract more helper methods in the future but I wanted to keep the changes in a reasonable scope for this PR that kept getting bigger :D